### PR TITLE
Creating Stable version from Preview version

### PR DESCRIPTION
**Pull request by navysingla**
In this PR, we are creating new version 2020-09-01 from 2019-10-01-preview version.
We are not adding all the APIs in stable version from preview version but only the subset of APIs from preview version.
Preview version has these APIs - 
1. Cancel SUb
2. Rename Sub
3. Enable Sub
4. Create Subscription Alias
5. Get Subscription Alias
6. List Subscription Alias
7. Delete Subscription Alias
8. Create Legacy Subscription
9. Create Modern Subscription
10. Create Csp Subscription
11. Get subscription Operation

We are adding only following APIs in stable version - 
1. Cancel SUb
2. Rename Sub
3. Enable Sub
4. Create Subscription Alias
5. Get Subscription Alias
6. List Subscription Alias
7. Delete Subscription Alias

### CODE REVIEW

File: specification/subscription/resource-manager/readme.md [hunk 1]
```diff
@@ -34,6 +34,18 @@ openapi-type: arm
 tag: package-2019-10-preview
```
Comment (fengzhou-msft) [inline specification/subscription/resource-manager/readme.md:34]: Would you like to change the default tag to `package-2020-09` as well?

Comment (navysingla) [reply specification/subscription/resource-manager/readme.md:34]: No, we don't it right now, once we are ready to generate SDKs from this version, I will send out another PR with that change

**Review by majastrz — APPROVED** [review]
Confirmed identical to `2019-10-01-preview`. Signed off from ARM side.

File: specification/subscription/resource-manager/readme.md (hunk 1 above)
Comment (navysingla) [reply specification/subscription/resource-manager/readme.md]: Changed it to new version

**Comment by navysingla** [comment]
@fengzhou-msft Not sure why sdk-net check is failing now, can you check ?

**Comment by navysingla** [comment]
> @fengzhou-msft Not sure why sdk-net check is failing now, can you check ?

@fengzhou-msft can you help here fixing sdk error, seems like some sdkautomation. And can we merge this PR

**Comment by fengzhou-msft** [comment]
@navysingla sdk-net failure can be ignored.

**Pull request merged**